conan/cli/command.py

```diff
@@ -83,19 +89,25 @@ def doc(self):
     def _format(self, parser, info, *args):
         parser_args, _ = parser.parse_known_args(*args)
 
-        default_format = "text"
-        try:
-            formatarg = parser_args.format or default_format
-        except AttributeError:
-            formatarg = default_format
+        formatarg = getattr(parser_args, "format", None) or "text"
+        out_file = getattr(parser_args, "out_file", None)
 
         try:
             formatter = self._formatters[formatarg]
         except KeyError:
             raise ConanException("{} is not a known format. Supported formatters are: {}".format(
                 formatarg, ", ".join(self._help_formatters)))
 
-        formatter(info)
+        if out_file:
+            with open(out_file, 'w') as f:
+                with redirect_stdout(f):
+                    formatter(info)
+            ConanOutput().info(f"Formatted output saved to '{out_file}'")
+        else:
+            # https://github.com/conan-io/conan/issues/17245 avoid colorama crash and overhead
+            colorama.deinit()
```

Comment:
This will be problematic if some formatter uses color. Note the previous location was conditioned to not ``if (fg or bg) and _color_enabled(sys.stdout):``